### ISSUE TITLE
Go to Definition not working on calls and types

### ISSUE THREAD

**Issue by yannickmoy** [dafny-lang/ide-vscode#20]
Using Go to Definition from the contextual menu on uses of type `Str` and function `Ident` on the following code, the plugin shows some activity for a few seconds and does nothing:

```
module Test {
    type Str = string

    function Ident(s: Str): Str {
        s
    }

    function Wrapper(s: Str): Str {
        Ident(s)
    }
}
```

I am using version 1.4.0 of the plugin with VSCode 1.52.1 on Darwin.

**Comment by camrein** [comment]
The resolution of type aliases is not yet supported. But it should do so for your `Ident` function. Could you please test the following example:

```dafny
method Main() {
  assert false;
}
```
If you didn't disable the automatic verification (either set it to *onsave* or *onchange*), it should underline the `false` literal. I have the feeling that you ran into the same issue reported in #16. Could you please double-check that you use the new [dafny language server](https://github.com/dafny-lang/language-server-csharp) instead of the old server that is shipped with the Dafny compiler? If the plugin correctly loads, you should see the message *DafnyLS: 3.0.0* on the bottom bar.

**Comment by yannickmoy** [comment]
Thanks @camrein !

I see _DafnyLS: 3.0.0_ in the bottom bar. That's strange, the "Go to Definition" on `Ident` now works, maybe it was just very slow. But if you turn `Ident` into a lemma, again the "Go to Definition" does not work:

```
module Test {
    type Str = string

    lemma Ident(s: Str) {
    }

    function Wrapper(s: Str): Str {
        Ident(s);
        s
    }
}
```

**Comment by camrein** [comment]
You may have observed a slow startup. I can confirm that the lemma, in this case, cannot be resolved. I have fixed it in dafny-lang/language-server-csharp/pull/47.

**Comment by yannickmoy** [comment]
thank you!
